### ISSUE TITLE
Add session attribute icons to Activities List

### ISSUE THREAD

**Issue by saschaben** [ilios/frontend#2912]
In order to provide students with an even more streamlined experience, let's unbury the labcoat/equipment/attendance icons from within the event page, and add them to the activities list on the dashboard:
![activitiesicons](https://cloud.githubusercontent.com/assets/1753431/25442817/e492638a-2a5a-11e7-807b-5ea8292c6b63.png)
every icon should have a mouseover title value:
**recently updated**: "_new updates_"
**special attire required**: "_whitecoats / special attire_"
**special equipment required**: "_special equipment_"
**attendance required**: "_attendance is required_"

**Comment by mtb3** [comment]
Hello, I wanted to jump in and mention that administrative users at the University of Florida have requested this feature, in particular for the **Attendance Required** attribute.

**Comment by jrjohnson** [comment]
@mtb3 coincidentally we added some code to the API yesterday that is going to make this easy to do. I added it to our 'ready' list and we should have some good news on this in a few weeks at most.

**Comment by mtb3** [comment]
@jrjohnson That sounds great; thanks for the update.
